### DISCUSSION TITLE
Welcome to My-Portfolio Discussions! Let's discuss random things...

### DISCUSSION THREAD

**Discussion by itsvijaysingh** [itsvijaysingh/My-Portfolio#1] [Announcements]
## 👋 Welcome!

This is where we can hang out, chat, and discuss anything related to my projects, suggestions for new ones, or just random things that spark our interest!

Feel free to drop your thoughts, ideas, or questions. Whether it's feedback on existing projects, suggestions for improvements, or brainstorming new ideas, your input is highly valued!

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kupendrav** [comment]
how long you been into coding?
how are you managing backend?
which tools did you use for your portfolio website?(its simply amazing)

**Reply by itsvijaysingh** [reply]
Welcome, **Kupendra bhai**! Thanks for joining and for your first message in this discussion section.

**How long have you been into coding?**
> I got serious about coding around July 2023 during my MCA studies(presently pursuing) and started coding again by learning **C**. But my coding journey began in 2021 with Discord bots. I was mostly copy-pasting code and learning **Discord.js**. Life got in the way for a bit, but I picked it back up in July 2023.

How are you managing the backend?
> For this portfolio, I've kept things relatively simple by focusing on the frontend aspects. However, for more complex projects, I typically manage backend technologies like Node.js with Express for server-side functionalities, along with databases like MongoDB for data management.

Which tools did you use for your portfolio website?
> Thank you for the compliment! I built this portfolio using **HTML, CSS, Bootstrap, **and some** JavaScript**. Additionally, I utilized libraries such as Owl Carousel for testimonials, jQuery for supporting Owl Carousel, and AOS for fade effects. The fonts used are Augustina, Rubik, and Dancing Script, along with Bootstrap Icons for some of the graphical elements.

You might also be asking about where I coded it or created the images:
I used **[VS Code](https://code.visualstudio.com/)** for coding and [Canva](https://canva.com) for creating images. I also used icons from **Dev Icons**. Regarding my portrait image, it was created with **[Remini AI](https://remini.ai/)**.
